QA warnings:   cmd/starter/c/starter.c:782:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]

Hi,

I get QA warnings with clang-20: 
```
 * cmd/starter/c/starter.c:782:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
 * cmd/starter/c/starter.c:796:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
 * cmd/starter/c/starter.c:810:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
 * ../cmd/starter/c/starter.c:782:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
 * ../cmd/starter/c/starter.c:796:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
 * ../cmd/starter/c/starter.c:810:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
```

--- a/cmd/starter/c/starter.c
+++ b/cmd/starter/c/starter.c
@@ -779,6 +779,7 @@ static int uts_namespace_init(struct namespace *nsconfig) {
         }
         return CREATE_NAMESPACE;
     }
+    return NO_NAMESPACE;
 }
 
 static int ipc_namespace_init(struct namespace *nsconfig) {
@@ -793,6 +794,7 @@ static int ipc_namespace_init(struct namespace *nsconfig) {
         }
         return CREATE_NAMESPACE;
     }
+    return NO_NAMESPACE;
 }
 
 static int cgroup_namespace_init(struct namespace *nsconfig) {
@@ -807,6 +809,7 @@ static int cgroup_namespace_init(struct namespace *nsconfig) {
         }
         return CREATE_NAMESPACE;
     }
+    return NO_NAMESPACE;
 }
 
 static int mount_namespace_init(struct namespace *nsconfig, Bool masterPropagateMount) {

